docs/Video Streaming for Navigation Apps/Touch Input/index.md

```diff
@@ -1,13 +1,20 @@
 # Touch Input
-Navigation applications have support for touch events, including both single and multitouch events. This includes interactions such as panning and pinch. A developer may use the included `SDLTouchManager` class, or yourself by listening to the `SDLDidReceiveTouchEventNotification` notification.
+Navigation applications have support for touch events, including both single and multitouch events. This includes interactions such as panning and pinch. A developer may use the included @![iOS]`SDLTouchManager`!@ @![android]`TODO android does not use touch manager so add proper information here`!@ class, or yourself by listening to the @![iOS]`SDLDidReceiveTouchEventNotification`!@ @![android]`TODO Add proper detail here for the event listener`!@ notification.
+
+@![android]
+`// TODO - See indx.md for example and add inLINE android tags to align the section.`
```

Comment:
Just use one TODO instead of a bunch

Reply:
So do you want me to remove the inLine Andoird tags and only leave the one at like 4

Reply:
Yeah. And fix it up to read better.

Reply:
changed